assets/js/sidebar/sidebar-drawer.js

```diff
@@ -136,13 +143,13 @@ export function openSidebar () {
   sessionStorage.setItem('sidebar_state', 'opened')
   qs(SIDEBAR_TOGGLE_SELECTOR).setAttribute('aria-expanded', 'true')
 
-  requestAnimationFrame(() => {
```

Comment:
`openSidebar` and `closeSidebar` did not actually return `Promise`s so I made a small update so that pressing `v` with a closed sidebar would pop open the sidebar and _then_ open/focus the version select.